src/coins.cpp

```diff
@@ -443,6 +443,18 @@ double CCoinsViewCache::GetPriority(const CTransaction &tx, int nHeight) const
             dResult += coins->vout[txin.prevout.n].nValue * (nHeight-coins->nHeight);
         }
     }
+
+    // Joinsplits do not reveal any information about a note's amount or age.
+    // We need to ensure priority increases the longer a tx remains in the mempool, so we don't
+    // generate random values which would change on each invocation of this method.
+    // Instead, we use the anchor as a seed to generate the note value deterministically.
+    // Currently, we say that each JoinSplit has input notes of between 1 to 2 ZEC,
+    // and that the age of each note is half the age of the blockchain.
```

Comment:
I would prefer we treat JoinSplits as having a fixed value. Otherwise, what stops a transaction creator from grinding over possible anchors to ensure that their transaction will be treated as higher priority than others?

I continue to be skeptical that making piecemeal changes to the fee/prioritisation policy relative to Bitcoin 0.11.2 is the right approach for Zcash; I think a redesign and drastic simplification is called for.